wodles/azure/tests/test_azure.py

```diff
@@ -708,6 +709,47 @@ def test_get_blobs(mock_send, mock_update, blob_date, min_date, max_date, desire
             [blob.name for blob in blob_list if extension in blob.name])
 
 
+@patch('azure-logs.update_row_object')
+@patch('azure-logs.send_message')
+def test_get_blobs_only_with_prefix(mock_send, mock_update):
+    """Test get_blobs obtains the blobs only with prefix from a container and send their content to the socket."""
```

Comment:
The test is not checking that events are sent to the socket. Change the description to something more accurated, such as this example: 
```suggestion
    """Test get_blobs process only the blobs corresponding to a specific prefix, ignoring the rest."""
```